Source/WebCore/platform/MediaStrategy.h

```diff
@@ -63,6 +63,8 @@ class WEBCORE_EXPORT MediaStrategy {
     virtual void nativeImageFromVideoFrame(const VideoFrame&, CompletionHandler<void(std::optional<RefPtr<NativeImage>>&&)>&&);
 #endif
 
+    virtual bool enableWebMMediaPlayer() const { return true; }
```

Comment:
Maybe wrap this up with `#if ENABLE(ALTERNATE_WEBM_PLAYER)` considering it's only used there.

bonus points for renaming `ALTERNATE_WEBM_PLAYER` to maybe `COCOA_WEBM_PLAYER`